src/main/java/com/exasol/sql/expression/function/exasol/ExasolFunction.java

```diff
@@ -14,7 +15,7 @@
     private static final List<String> functionsWithoutParenthesis = Arrays.asList("SYSDATE", "CURRENT_SCHEMA",
```

Comment:
I don't like that you don't use Enum here.  Maybe at least SYSDATE.toString()

Reply:
There was a reason why I used Strings instead of enum. Something didn't work, But I don't remember what :D